tensorflow/python/ops/ctc_ops.py

```diff
@@ -31,7 +31,8 @@
 def ctc_loss(labels, inputs, sequence_length,
              preprocess_collapse_repeated=False,
              ctc_merge_repeated=True,
-             ignore_longer_outputs_than_inputs=False, time_major=True):
+             ignore_longer_outputs_than_inputs=False, time_major=True,
+             name=None):
```

Comment:
The added `name` kwarg needs to be documented in the Args section of the doc string below.